spec/controllers/user_profile/about_me_controller_spec.rb

```diff
@@ -188,6 +188,61 @@
 
     end
 
+    context 'with spam attributes and a non-whitelisted user' do
+
+      let(:user) { FactoryGirl.create(:user, :name => 'JWahewKjWhebCd') }
+
+      before :each do
+        UserSpamScorer.spam_score_threshold = 1
+        UserSpamScorer.score_mappings =
+          { :about_me_includes_currency_symbol? => 20 }
+        session[:user_id] = user.id
+      end
+
+      after(:each) { UserSpamScorer.reset }
```

Comment:
Ah, that's GitHub borking the commit order. Here's what it looks like in my terminal:

```
alaveteli 3301-proactive-spam-checker@ ll
97e9bab Freeze mutable objects assigned to constants (Gareth Rees)
38a77c1 Prevent users setting their about_me as spam (Gareth Rees)
b4751eb Add UserSpamScorer#spam? (Gareth Rees)
2785251 Add UserSpamScorer spam score threshold (Gareth Rees)
ad61d02 Use class attributes to set default instance values (Gareth Rees)
49d6c2d Add UserSpamScorer class attribute accessors (Gareth Rees)
ce67b42 Add missing spec (Gareth Rees)
c5ecbdc Replace UserController#set_profile_about_me (Gareth Rees)
43de19d Validate User#about_me in model (Gareth Rees)
```